Sources/MapboxNavigation/InstructionsCardViewController.swift

```diff
@@ -169,6 +171,14 @@ open class InstructionsCardViewController: UIViewController {
         }
     }
     
+    open func updateCurrentVisibleInstructionCard(for instruction: VisualInstructionBanner) {
+        guard let remainingStepsCount = routeProgress?.currentLegProgress.remainingSteps.endIndex else { return }
+        let indexPath = IndexPath(row: 0, section: 0)
+        if let container = instructionContainerView(at: indexPath), indexPath.row < remainingStepsCount {
```

Comment:
Should we reorder conditions like this?
```suggestion
        if indexPath.row < remainingStepsCount, let container = instructionContainerView(at: indexPath) {
```

Reply:
Thanks for the suggestion, I'll improve this part.